e2e/integration/helpers/helpers.go

```diff
@@ -0,0 +1,341 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/kubeedge/kubeedge/e2e/integration/utils"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
+)
+
+const (
+	podLabelSelector = "?labelSelector="
+)
+
+func SenHttpRequest(method, reqapi string) (error, *http.Response){
```

Comment:
Pls add a comment with a short description explaining the functionality of this function

Reply:
nit: SenHttpRequest -> SendHttpRequest ?